app/res/layout/entity_select_layout.xml

```diff
@@ -3,13 +3,25 @@
   xmlns:android="http://schemas.android.com/apk/res/android"
   android:layout_height="wrap_content" android:layout_width="fill_parent">
 
-
+<TextView
+    android:id="@+id/no_search_results"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:layout_below="@+id/universal_frame_tile"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
```

Comment:
This should either be a style or at least

android:padding: FULL_PADDING_TEXT

or something, right?